src/js/screen_gfx.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
-const fontFamily = '"sans-serif-condensed", "Arial Narrow", sans-serif';
+const fontFamily =
+  '"sans-serif-condensed", "Arial Narrow", "Roboto Condensed", "Noto Sans Armenian", sans-serif';
 export const shieldFont = (size) => `bold ${size}px ${fontFamily}`;
```

Comment:
Add this and it actually works on firefox android

```suggestion
export const shieldFont = (size) => `condensed bold ${size}px ${fontFamily}`;
```